### ISSUE TITLE
.NET SDK emits NETSDK1206 warning in .NET 8.0

### ISSUE THREAD

**Issue by zeroskyx** [Azure-Samples/cognitive-services-speech-sdk#2178]
Greetings,

when using the current version of the [.NET SDK](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech), compiling a solution causes [NETSDK1206](https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/netsdk1206) because the package references platform-specific components:

`warning NETSDK1206: Found version-specific or distribution-specific runtime identifier(s): centos7-x64, win10-arm, win10-arm64. Affected libraries: Microsoft.CognitiveServices.Speech. In .NET 8.0 and higher, assets for version-specific and distribution-specific runtime identifiers will not be found by default. See https://aka.ms/dotnet/rid-usage for details.`

Is it feasible to remove these specific artifacts in the next version, to get rid of the warning?

Thanks in advance and all the best

**Comment by pankopon** [comment]
Duplicate of #2170

**Comment by zeroskyx** [comment]
Hi @pankopon and thanks for looking into this.
Unfortunately, the issue has NOT been resolved in `1.34.0` as that is the version that I'm using.
Did the fix not work, or might this be another issue here?

Thanks again

**Comment by pankopon** [comment]
@zeroskyx Apologies for the confusion. There was a partial fix so that warnings about `win10-arm` and `win10-arm64` are removed (the ids were changed to `win-arm` and `win-arm64`) but the warning about `centos7-x64` remains. This is due to CentOS 7 that we still have to continue to support with specific binaries. You just have to ignore the remaining warning. We will update documentation at some point to reflect this.

To verify the change in warnings, make sure you are indeed using 1.34.0 packages. For example, using the sample
https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnetcore/console
change `TargetFramework` from `netcoreapp3.1` to `net8.0` and build.

With 1.33.0 packages + Visual Studio 2022 version 17.8.3:
```
1>------ Build started: Project: samples, Configuration: Release Any CPU ------
1>C:\Program Files\dotnet\sdk\8.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(284,5): warning NETSDK1206: Found version-specific or distribution-specific runtime identifier(s): centos7-x64, win10-arm, win10-arm64. Affected libraries: Microsoft.CognitiveServices.Speech, Microsoft.CognitiveServices.Speech.Extension.MAS. In .NET 8.0 and higher, assets for version-specific and distribution-specific runtime identifiers will not be found by default. See https://aka.ms/dotnet/rid-usage for details.
```
With 1.34.0 packages:
```
1>------ Build started: Project: samples, Configuration: Release Any CPU ------
1>C:\Program Files\dotnet\sdk\8.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(284,5): warning NETSDK1206: Found version-specific or distribution-specific runtime identifier(s): centos7-x64. Affected libraries: Microsoft.CognitiveServices.Speech, Microsoft.CognitiveServices.Speech.Extension.MAS. In .NET 8.0 and higher, assets for version-specific and distribution-specific runtime identifiers will not be found by default. See https://aka.ms/dotnet/rid-usage for details.
```
